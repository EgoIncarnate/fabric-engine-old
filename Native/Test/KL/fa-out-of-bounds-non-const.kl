operator entry()
{
  var Integer a[2];
  a[0] = 17;
  a[1] = 42;
  a[2] = 64;
}
