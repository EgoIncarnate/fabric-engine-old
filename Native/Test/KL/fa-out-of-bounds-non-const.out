KL: Integer[2] index out-of-bounds
